pkg/config/app.go

```diff
@@ -255,6 +257,31 @@ func (c *AppConfig) setEnvVariableConfig() {
 	viper.AutomaticEnv()
 }
 
+// pulls version from git context
+func getVersion() string {
+    cmd := exec.Command("git", "describe", "--tags", "--abbrev=0")
```

Comment:
that's not the correct way to do it - the version have to be static into the code, but gathered dynamically during build time.

see this as an example

Reply:
Noted, will go back and review.

Reply:
Version is now being injected and set during build.  Let me know if it needs further revision.